src/patternfly/components/DataToolbar/data-toolbar-expandable-content.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="pf-c-data-toolbar__expandable-content{{#if data-toolbar-expandable-content--IsExpanded}} pf-m-expanded{{/if}}{{#if data-toolbar-expandable-content--modifier}} {{data-toolbar-expandable-content--modifier}}{{/if}}" id="{{data-toolbar--id}}"
```

Comment:
can you put `id="{{data-toolbar--id}}"` on a new line so it's before `{{#if data-toolbar-expandable-content--attribute}}`?